### ISSUE TITLE
LXC 1.1.3

### ISSUE THREAD

**Issue by aarnaud** [lxc/python2-lxc#6]
Hi,

I use the  lxc stable ppa on Ubuntu trusty. Since LXC 1.1.3 i can't create container:

The template args failed,

````
# ps ax | grep 17958
/bin/sh /usr/share/lxc/templates/lxc-download --path=/var/lib/lxc/ubuntu-trusty-amd64-default --name=ubuntu-trusty-amd64-default --rootfs=/var/lib/lxc/ubuntu-trusty-amd64-default/rootfs  н??L?
````
The source arguments is `{"dist": "ubuntu", "release": "trusty", "arch": "amd64", "variant": "default"}`
The final arguments is `н??L?`

Thanks,

**Comment by stgraber** [comment]
If you're using python2-lxc, did you rebuild it after upgrading to 1.1.3?

As listed in the announcement, LXC 1.1.3 fixes an ABI regression which happened with 1.1.0, which means anything which was built on older LXC versions must be rebuilt.

**Comment by aarnaud** [comment]
Thanks this is fix with a reinstall
But i couldn't uninstall lxc-python2 properly with `pip uninstall`, i used rm command.

````
~# pip2 uninstall lxc-python2
Can't uninstall 'lxc-python2'. No files were found to uninstall.
~# rm  -Rf /usr/local/lib/python2.7/dist-packages/lxc_python2-0.1-py2.7.egg-info
~# pip2 install lxc-python2
Downloading/unpacking lxc-python2
  Downloading lxc-python2-0.1.zip
  Running setup.py (path:/tmp/pip_build_root/lxc-python2/setup.py) egg_info for package lxc-python2
    
Installing collected packages: lxc-python2
  Running setup.py install for lxc-python2
    building '_lxc' extension
    x86_64-linux-gnu-gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fPIC -I/usr/include/python2.7 -c lxc.c -o build/temp.linux-x86_64-2.7/lxc.o
    x86_64-linux-gnu-gcc -pthread -shared -Wl,-O1 -Wl,-Bsymbolic-functions -Wl,-Bsymbolic-functions -Wl,-z,relro -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -D_FORTIFY_SOURCE=2 -g -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security build/temp.linux-x86_64-2.7/lxc.o -llxc -o build/lib.linux-x86_64-2.7/_lxc.so
    
Successfully installed lxc-python2
Cleaning up...
````
